### ISSUE TITLE
Ça va ?

### ISSUE THREAD

**Issue by JeffAbrahamson** [Killy85/MachineLearningExercises#5]

**Comment by Killy85** [comment]
Yes, everything is ok for me. I may miss some documentation on some parts of the code. I'll try to get that right and i'll come back to you with issues.
